bundles/framework/coordinatetool/plugin/helper.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Transforms the given coordinates using action_route=Coordinates and updates coordinates to the UI
+ * @method getTransformedCoordinatesFromServer
+ * @param {Object} data: {lonlat: lat: '', lon: ''} coordinates to be transformed
+ * @param {String} srs: projection for given lonlat params like "EPSG:4326"
+ * @param {String} targetSRS: projection to transform to like "EPSG:4326"
+ * @param {Function} successCb success callback
+ * @param {Function} errorCb error callback
+ */
+export const getTransformedCoordinatesFromServer = async (mapModule, data, srs, targetSRS) => {
```

Comment:
Looks like this isn't used anywhere, but if it is we should pass the mapModule.getProjection() as param instead of the whole mapModule since it's the only thing that is used from mapModule here.